helm/defectdojo/templates/network-policy.yaml

```
@@ -9,26 +9,29 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     helm.sh/chart: {{ include "defectdojo.chart" . }}
     app.kubernetes.io/name: {{ include "defectdojo.name" . }}
+    {{- with .Values.extraLabels }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
 spec:
   podSelector:
     matchLabels:
       app.kubernetes.io/instance: {{ .Release.Name }}
-  {{- if .Values.networkPolicy.ingress}}
+  {{- with .Values.networkPolicy.ingress }}
```

I don't think `with` will work in this case since there is an `else` on line 22 that depends on this `if`

```suggestion
  {{- if .Values.networkPolicy.ingress }}
```